configyamls/all/otel-config.yaml

```diff
@@ -230,12 +234,13 @@ service:
       exporters:
       - otlp/2
       processors:
+      - resource/traces
       - resourcedetection
-      - resource/3
+      - resource/hostid
       - attributes/traces
       - batch
       receivers:
       - otlp
   telemetry:
     logs:
-      level: fatal
+      level: fatal
```

Comment:
new line char is missing at end of the file.

do where ever applicable in this PR

Reply:
added newline back.